packages/tasks/src/tasks/text-to-video/data.ts

```diff
@@ -68,15 +68,15 @@ const taskData: TaskDataCustom = {
 	models: [
 		{
 			description: "A strong model for consistent video generation.",
-			id: "rain1011/pyramid-flow-sd3",
+			id: "tencent/HunyuanVideo",
 		},
 		{
-			description: "A robust model for text-to-video generation.",
-			id: "VideoCrafter/VideoCrafter2",
+			description: "A text-to-video model with high fidelity motion and strong prompt adherence.",
+			id: "genmo/mochi-1-preview",
```

Comment:
```suggestion
			id: "Lightricks/LTX-Video",
```

Quite powerful and the community loves it, Mochi-1 is a bit too VRAM hungry

Reply:
this page is for text-to-video, that model is added to image-to-video already

Reply:
But, LTX-Video is Text to Video: https://huggingface.co/Lightricks/LTX-Video 😅

Reply:
most of the models that are image-to-video are actually image+text-to-video, so most of the time the differentiator here is the fact that it can take in an image as initial frame or video as few initial frames. I have no strong opinion here but I find it confusing it redirects to a model tagged as image-to-video